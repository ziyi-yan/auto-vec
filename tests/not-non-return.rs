use auto_vec::auto_vec;

#[auto_vec]
fn foo(i: i32) {}

fn main() {}
