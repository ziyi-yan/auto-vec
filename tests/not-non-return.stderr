error: expected a fn with returned type
 --> $DIR/not-non-return.rs:3:1
  |
3 | #[auto_vec]
  | ^^^^^^^^^^^
